pkg/forwarder/forwarder.go

```diff
@@ -201,7 +206,7 @@ func NewDefaultForwarder(options *Options) *DefaultForwarder {
 			log.Errorf("No API keys for domain '%s', dropping domain ", domain)
 		} else {
 			f.keysPerDomains[domain] = keys
-			f.domainForwarders[domain] = newDomainForwarder(domain, options.NumberOfWorkers, options.RetryQueueSize)
+			f.domainForwarders[domain] = newDomainForwarder(domain, options.NumberOfWorkers, options.RetryQueueSize, options.ConnectionResetInterval)
```

Comment:
Any reasons to fetch `forwarder_connection_reset_interval` in the Forwarder and pass it as a parameter instead of fetching is in the domain_forwarder directly ?

Reply:
the `domain_forwarder` currently doesn't import the `config` package, and I prefer to keep it that way (makes it easier to test and use from other packages IMO). Also the forwarder's `Options` struct felt like a perfect fit for this new option.